apps/keyboard/js/keyboard.js

```diff
@@ -1525,7 +1525,18 @@ function sendKey(keyCode, isRepeat) {
 
   default:
     if (inputContext) {
-      return inputContext.sendKey(0, keyCode, 0);
+      // to match the behavior on desktop Firefox, set upperKeyCode as
+      // keyCode for digits
+      console.log(upperKeyCode, keyCode);
```

Comment:
Sorry that I missed this comment. This PR has been reverted and another corrected one will be merged instead.